<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_chartTimeXYWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gantt</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="GanttWidget" name="ganttWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_ganttPlayer">
            <property name="text">
             <string>Плеер</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ganttClear">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ganttAddItems">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ChartXYWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="ChartXYWidget" name="widget_chartXYWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ChartTimeXYWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="IntervalSlider" name="widgetIntervalSlider" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ChartTimeXYWidget" name="widget_chartTimeXYWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_showLegend">
            <property name="text">
             <string>ShowLegend</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>ScaleDivChange</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>ChartsGroupWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="ChartsGroupWidget" name="widget_chartsGroupWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ChartTimeXYGroupWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
        <item>
         <widget class="ChartTimeXYGroupWidget" name="widget_chartXYGroupWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_syncAxisX">
            <property name="text">
             <string>Общая ось X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>ClearSelection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="MemoryPlanningWidget" name="memoryPlanningWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>DtIntervalWidget</string>
       </attribute>
       <widget class="DtIntervalWidget" name="dtIntervalWidget" native="true">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>601</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MemoryPlanningWidget</class>
   <extends>QWidget</extends>
   <header>memoryplanningwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IntervalSlider</class>
   <extends>QWidget</extends>
   <header>intervalslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttWidget</class>
   <extends>QWidget</extends>
   <header>ganttwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartTimeXYWidget</class>
   <extends>QWidget</extends>
   <header>charttimexywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartTimeXYGroupWidget</class>
   <extends>QWidget</extends>
   <header>chartsgroupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DtIntervalWidget</class>
   <extends>QWidget</extends>
   <header>dtintervalwidget/dtintervalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartsGroupWidget</class>
   <extends>QWidget</extends>
   <header location="global">chartsgroupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartXYWidget</class>
   <extends>QWidget</extends>
   <header location="global">chartxywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
